kura/org.eclipse.kura.web2/pom.xml

```diff
@@ -252,7 +252,7 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.3</version>
 				<configuration>
-					<source>1.6</source>
+					<source>1.7</source>
```

Comment:
How do you come to this conclusion from this document? What I read is that the source is validated against Java 6 rules:

> Specifies the version of source code accepted. The following values for release are allowed:
> …
> No language changes were introduced in Java SE 6. However, encoding errors in source files are now reported as errors instead of warnings as in earlier releases of Java Platform, Standard Edition.

Where the resulting byte code would be Java 7:

> Generates class files that target a specified release of the virtual machine. Class files will run on the specified target and on later releases, but not on earlier releases of the JVM. 


Which, in that combination, doesn't make any sense to me.